### PR TITLE
1.1b

### PR DESCRIPTION
**Pull request by Sav22999**
- Updated English strings (thanks @cloudrac3r)
- Fixed some bugs
- Improved something
- Added "Daily goal" #35 
- Added check of new version #56 
- Support for Turkish (tr)

### CODE REVIEW

**Comment by Sav22999** [comment]
*I'm waiting for the translations*

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -166,8 +170,10 @@
     <string name="description_setting_daily_goal">Choose how many sentences you want to record and clips you want to validate (sum of them) per day.</string>
     <string name="daily_goal_achieved_today">Achieved today</string>
     <string name="daily_goal_not_achieved_today">Not achieved today</string>
-    <string name="daily_goal_achieved_message">Well done! You achieved your daily goal.\nYou validated {{*{{n_clips}}*}} and recorded {{*{{n_sentences}}*}}.\nFeel free to continue contributing to Common Voice.</string>
+    <string name="daily_goal_achieved_message">Well done! You achieved your daily goal.\nYou validated {{*{{n_clips}}*}} clip(s) and recorded {{*{{n_sentences}}*}} sentence(s).\nFeel free to continue contributing to Common Voice.</string>
     <string name="daily_goal_is_not_official_by_mozilla">This daily goal isn\'t an official Mozilla feature. It\'s only a feature of this app.</string>
```
Comment (cloudrac3r) [inline app/src/main/res/values/strings.xml:174]: F-Droid should have a capital D.

Comment (Sav22999) [reply app/src/main/res/values/strings.xml:174]: Absolutely true, thanks

Comment (cloudrac3r) [reply app/src/main/res/values/strings.xml:174]: Github did not select the right thing.

**Comment by cloudrac3r** [comment]
👍

**Pull request merged**